en/docs/administer/managing-users-and-roles/managing-user-roles.md

```diff
@@ -138,11 +138,15 @@ From APIM 3.1.0 onwards, we have introduced a `Role Mapping` feature to enable u
 
 1. Log in to the admin portal ( `https://<APIM_Host>:<APIM_Port>/admin` ) if you have not done already.
 
-2. Navigate to Settings > Scope Mapping in admin portal.
+2. Navigate to Settings > Role Permissions in admin portal.
 
-3. Under Role Mappings define a mapping as follows.
+3. Click **Add role permission** and provide a role name created in a carbon console. Then click **Next** to proceed.
 
-![Add Role Mapping]({{base_path}}/assets/img/administer/role-mapping.png)
+    ![Add Role Mapping]({{base_path}}/assets/img/administer/add-new-role-mapping.png)
+
+4. Under Select Permissions, click Role alias to define a mapping.
```

Comment:
what is role alias? is it different to the role name? or is it the scope name?

Reply:
Here the alias means the role that we need need to map for the new role.